### PR TITLE
new: Add hi (Hindi) language.

### PR DESCRIPTION
**Pull request by sudipshil9862**
adding hindi language support for gtkemojichooser

### CODE REVIEW

**Comment by sudipshil9862** [comment]
@milesj  please review my PR. all checks have passed

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
     strategy:
       matrix:
         locale:
+          - hi
```
Comment (milesj) [inline .github/workflows/build.yml:43]: can you alphasort this, and in the other files too

File: po/hi/messages.po [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+"X-Generator: Poedit 3.3.2\n"
+
+#, javascript-format
+msgctxt "LABEL: Denotes a countries flag using alphabetical characters (usa = United States)"
+msgid "regional indicator %s"
+msgstr "আঞ্চলিক সূচক %s"
+
+msgctxt "LABEL: Very short word for skin tone and/or color (max 1 word if possible)"
+msgid "tone"
+msgstr ""
```
Comment (milesj) [inline po/hi/messages.po:24]: all of these need to be translated

**Review by milesj — CHANGES_REQUESTED** [review]
Why was `packages/data/en/shortcodes/joypixels.raw.json` deleted?

**Comment by sudipshil9862** [comment]
@milesj  please review my PR. all checks have passed

**Comment by sudipshil9862** [comment]
@milesj please merge it if everything is fine as I need to make PR on gtkemojichooser

**Comment by milesj** [comment]
Awesome

**Pull request merged**